src/assets/css/local/search.css

```diff
@@ -0,0 +1,33 @@
+form#search {
+  .filter-and-results {
+    display: flex;
+    flex-direction: row;
+
+    &.filter-and-results--hidden {
+      display: none;
+    }
+
+    fieldset.types {
+      display: flex;
+      flex-direction: column;
+      width: fit-content;
+      font-size: smaller;
+    }
+
+    .results-area {
+      padding-right: 0.7em;
```

Comment:
`0.7em` is a bit of a magic number, it makes the search results line up nicely with the search input. I tried to work out where it was coming from and set it more nicely, but could find it :/